app/src/main/java/com/yalta/activities/fragments/MapFragment.kt

```diff
@@ -52,6 +57,10 @@ class MapFragment : Fragment(), OnMapReadyCallback {
 
         binding.lifecycleOwner = this
         binding.viewModel = viewModel
+        binding.viewModel?.points?.observe(viewLifecycleOwner, { points ->
+            this.points = points
```

Comment:
yeah, no, points should not be in fragment

Reply:
well, now these are another points